tests/e2e/Services/Webhooks/WebhooksBase.php

```diff
@@ -296,6 +297,7 @@ public function testUpdateDocument(array $data): array
     /**
      * @depends testCreateCollection
      */
+    #[FlakyTest(retries: 1)]
```

Comment:
I love the idea of adding this! Is this type of syntax common? Or are docblock tags more common? When looking into this, did you take a look at [bshaffer/phpunit-retry-annotations](https://github.com/bshaffer/phpunit-retry-annotations)?

Reply:
I think annotations are likely far more common as attributes were only added in PHP 8. However, attributes are objectively better, check out why in [the RFC here](https://wiki.php.net/rfc/attributes_v2#why_not_extending_doc_comments).

 I did not, I found a retry trait example on some Drupal forum and modified it to fit. Basically the same thing though